templates/content/call-for-papers-track-2.md

```diff
@@ -0,0 +1,48 @@
+<center> **Track 2: Applications and Practice: Investigation, Evaluation, Interpretation, and Deployment** </center>
+
+### Track Chairs
+- Dr. Tom Pollard
+- Dr. Bobak Mortazavi
+- Dr. Andrew Beam
+- Dr. Uri Shalit
+
+<br />
+
+### Description
+The goal of this track is to highlight works applying robust methods, models, or practices to identify, characterize, audit, evaluate, or benchmark systems. Additionally, unique deployments and datasets used to empirically evaluate these systems are necessary and important to advancing practice, Whereas the goal of [Track 1](/call-for-papers.html#tab-track-1) is to select papers that show significant technical novelty, submit your work here if the contribution is either more focused on solving a carefully motivated problem grounded in applications or on deployments or datasets that enable exploration and evaluation of applications. Introducing a new method is not prohibited by any means for this track, but the focus should be on methods which are designed to work particularly robustly (e.g., fail gracefully in practice), scale particularly well either in terms of computational runtime or data required, work across real-world data modalities and systems, etc. Contributions will be evaluated for technical rigor, robustness, and comprehensivity. We encourage applications and practice in both traditional and emerging clinical areas (e.g., models in electronic health records as well as applications in emerging fields such as remote and telehealth, integration of omics, etc.)
+
+Computer software submissions should describe the intended use for the software, justify the need for the software, and provide executable examples for other researchers. Software submissions should directly support a healthcare application. Examples include code for summarizing the demographics of a study cohort [4], deriving meaningful clinical concepts from electronic health records [5], and natural language processing tools specifically designed for clinical text [6, 7]. All computer software submissions must be open source and released under a suitable open source license. Computer software should adhere to best practices in software development where possible, including the use of unit tests, continuous integration, and diligent documentation of component design and purpose [8].
+
+4 Pollard TJ, Johnson AE, Raffa JD, Mark RG. tableone: An open source Python package for producing summary statistics for research papers. JAMIA Open. 2018 May 23;1(1):26-31.
```

Comment:
The formatting around these references is off, should be `[4] Pollard...` I think? And I guess it should start at 1.

Reply:
Content updated.